app/assets/stylesheets/components/_location-collection-item.scss

```
@@ -0,0 +1,38 @@
+.location-collection-item {
+  max-width: 64ex;
+  list-style-type: none;
+  padding-left: 0;
+  align-items: flex-start;
+  border-bottom-width: 1px;
+  border-bottom-style: solid;
+  display: flex;
+  margin-bottom: 1rem;
+  margin-top: 1rem;
+  padding-bottom: 1rem;
+  border-color: #cedced;
```

I think we have a variable for this, is it available here?
```suggestion
  border-color: $border-color;
```

found with:
```
> git grep cedced
app/assets/stylesheets/variables/_app.scss:$border-color: #cedced !default;
```

---

fixed in 235b440f0ec902eab3c13dba1ca539c91e0e5c54